e2e/utils.go

```diff
@@ -28,8 +28,10 @@ const (
 	defaultNs     = "default"
 	vaultSecretNs = "/secret/ceph-csi/"
 
-	rookTolBoxPodLabel = "app=rook-ceph-tools"
-	rbdmountOptions    = "mountOptions"
+	rookTolBoxPodLabel        = "app=rook-ceph-tools"
```

Comment:
You might want to send an other PR or add a commit that corrects the name of `ToolBox`. Not required, as you did not introduce the typo :)